packages/contentstack-import/src/app.js

```diff
@@ -87,15 +87,17 @@ let singleImport = async (moduleName, types, config) => {
       let exportedModule = require('./lib/import/' + moduleName);
       exportedModule
         .start(config)
-        .then(async function () {
+        .then(async function (data) {
           if (moduleName === 'content-types') {
             let ctPath = path.resolve(config.data, config.modules.content_types.dirName);
             let fieldPath = path.join(ctPath + '/field_rules_uid.json');
             if (fieldPath && fieldPath !== undefined) {
               await util.field_rules_update(config, ctPath);
             }
           }
-          addlogs(config, moduleName + ' imported successfully!', 'success');
+          if (!data) {
```

Comment:
@abhinav-from-contentstack Can you check for the specific key to check the condition? instead of **if (!data)** check **if (!(data && data.empty))**.

the reason is basically it's returning a promise of unknown data all we know is it's a promise<unknown|any> data. In the feature, if someone wants to return any data for any of the functionality it'll break the above code/condition.

CC @netrajpatel